### PR TITLE
Added guidance for fixing session start

### PR DESCRIPTION
**Pull request by meker12**
# Purpose of this pull request

Added guidance for fixing session start failures when launching Magento cloud
Docker on Windows environment

## Affected DevDocs pages

- https://devdocs.github.com/guides/v2.3/cloud/docker/docker-config.html

### CODE REVIEW

File: guides/v2.2/cloud/docker/docker-config.md [hunk 1]
```diff
@@ -130,20 +130,20 @@ Continue launching your Docker environment in the default _production_ mode.
...
 
-   -  Build Magento in the Docker container:
+   Build Magento in the Docker container:
 
-      ```bash
-      docker-compose run build cloud-build
-      ```
+   ```bash
+   docker-compose run build cloud-build
+   ```
```
Comment (meker12) [inline guides/v2.2/cloud/docker/docker-config.md]: @jeff-matthews  Question about the lint checking-- when you have code samples in nested list items, the linter fails with this error  `MD046 Code block style`.  Is there a work around for this?

Comment (jeff-matthews) [reply guides/v2.2/cloud/docker/docker-config.md]: @meker12, rule `MD046` is not enabled in the production style config, so I'm curious where you saw the error. Which build are you referring to? Neither the Jenkins nor the GitHub Actions Checks builds should fail on this rule.

I think there is a workaround though. I vaguely recall encountering something similar a while back. I'll have to do some digging. In the meantime, you should be able to use the bullets without failing production builds.

Comment (meker12) [reply guides/v2.2/cloud/docker/docker-config.md]: Thanks for the info -- I just got the error on my local machine, and was trying to resolve before merging to GitHub.

Comment (jeff-matthews) [reply guides/v2.2/cloud/docker/docker-config.md]: @dshevtsov, let us know if you remember a workaround for this.

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**